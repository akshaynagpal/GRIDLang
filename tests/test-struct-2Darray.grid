Player good
{ 
int x; 
int y; 
string s; 
} 

int gameloop()
{ 
Player good p1;
Player good(2,2) parray; 
p1.x = 3;
p1.y = 4;
p1.s = "2D array of structs work";
parray(0,0) = p1;
print(parray(0,0).x);
print(parray(0,0).y);
print(parray(0,0).s);
return 0; 
}
