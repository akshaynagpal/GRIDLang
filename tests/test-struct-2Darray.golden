3
4
2D array of structs work